### PR TITLE
New solutions partner: Goldbach Interactive

### PR DESCRIPTION
**Pull request by kellyfinn**

### CODE REVIEW

**Comment by kellyfinn** [comment]
This is ready @stewsmith @julieritchie

**Comment by stewsmith** [comment]
LGTM

File: website/partners/solutions/goldbach-interactive/index.hbs [hunk 1]
```diff
@@ -0,0 +1,34 @@
...
+    state: "Baden-Württemberg"
+    address1: "Bleicherstraße 10"
+    address2:
+    zip: "78467"
+    country: "Germany"
+contact:
+languages:
+- "German"
+- "English"
+tags:
```
Comment (stewsmith) [inline website/partners/solutions/goldbach-interactive/index.hbs]: @kellyfinn Make sure you use `TR_tags`

File: website/partners/solutions/goldbach-interactive/index.hbs [hunk 2]
```diff
@@ -0,0 +1,34 @@
...
+title: "Goldbach Interactive"
+solutions: true
+TR_summary: 
+logo: "//d1qmdf3vop2l07.cloudfront.net/optimizely-marketer-assets.cloudvent.net/raw/partner-logos/solutions/goldbach-international.png"
+website_link: "https://www.goldbachinteractive.de/"
+website_display: "www.goldbachinteractive.de"
+kb_article:
+stars: 1
+TR_industry: "Executive Consulting, Strategic Consulting, Custom Implementation, Test Development, Execution, & Management"
+locations:
```
Comment (stewsmith) [inline website/partners/solutions/goldbach-interactive/index.hbs]: @kellyfinn This also needs to be `TR_locations`

**Comment by stewsmith** [comment]
@kellyfinn I changed the `tags` to `TR_tags`. Just make sure you use `TR_tags` in the future. Thanks!

**Pull request merged**